src/frontend/admin/src/services/api/models/Organization.ts

```diff
@@ -19,6 +19,20 @@ export type Organization = {
   contact_phone?: string;
   contact_email?: string;
   dpo_email?: string;
+  addresses?: OrganizationAddress[];
+};
+
+export type OrganizationAddress = {
+  id: string;
+  title: string;
+  address: string;
+  postcode: string;
+  city: string;
+  country: string;
+  first_name: string;
+  last_name: string;
+  is_main: boolean;
+  is_reusable: boolean;
```

Comment:
In the case where the address is created, you should set those flag to `true`. (By default reusable is set to false in database).